examples/config.yml

```diff
@@ -0,0 +1,78 @@
+### Keys used by sio2:
+
+# Title of the package is defined in `title` key.
+title: Example task package
+# Title can be different for different languages.
+# They are defined in `title_<lang>` keys.
+# <lang> is a language code (for example 'pl' or 'en').
+# Available languages are defined in settings of `oioioi` instance.
+title_pl: Przykładowe zadanie
+
+# Number of points for each group can be defined in `scores` key.
+# If this key is not specified, then all groups have the same number of points.
+# (if number of groups doesn't divide 100, then the last groups will have the remaining points).
+# Group 0 always has zero points.
+scores:
+  1: 20
+  2: 30
+  3: 25
+  4: 25
+
+# Time limit for all tests is defined in `time_limit` key.
+# More precise time limit for each group or test can be defined in `time_limits` key.
+# The more precise time limit has higher priority (first test, then group, then global time limit).
+# Time limit is in milliseconds.
+time_limit: 1000
+time_limits:
+  1: 2000
+  1a: 2500
```

Comment:
Hmm, I think we don't want the functionality to specify the time limit on a particular test. Now we have a PDF during the competition which describes what is the exact time limit on a given group. Setting different time limits in a group is against this idea.

Reply:
In short, I believe that setting time limit on a test is a bad practice and we shouldn't allow the user to do so.

Reply:
I agree that this is a bad practice, but we can't just delete it. But I think we could write that there should be no reason for using this.

Reply:
Why we can't just delete it?

Reply:
It's not like we'll remove it from sio2, just from sinol-make.

Reply:
Entirely from the code or just from the example config?

Reply:
I would say that it's safer for the users to have this feature removed entirely from sinol-make. It's also not really good to have functionality that is not documented (and we don't really want to document this functionality).

Reply:
Now `sinol-make` exits when there is a time or memory limit for one test